libs/isograph-react/src/core/read.ts

```diff
@@ -797,6 +798,104 @@ export function readLinkedFieldData(
     };
   }
   const targetId = link;
+  if (typeof field.refetchQuery === 'number') {
```

Comment:
nit: `refetchQuery != null` I think is more idiomatic

can you add a comment saying `// if refetchQuery is not null, then the field is a client pointer`. Can you also add that to `ReaderLinkedField`?